### ISSUE TITLE
Template DBC file malformed

### ISSUE THREAD

**Issue by bit-dream** [stefanhoelzl/canpy#2]
Just wanted to point out a few bugs I found while using the template dbc file provided. I know this project isn't really actively maintained, but I did stumble upon it in my own investigation and ended up using the template DBC file for testing. So thought it still might be worthwhile to point out the few things that are incorrect.

As a preface, I consider the application CANDB++ the gold standard in terms of determining whether a dbc file is malformed or not, considering Vector is the one who implemented the standard (and it being their application). If you try to load the file into CANDB++, it will throw a parse error.

A few of the surface issues I found:
1. Bus speed has been deprecated since at least 2007 (BS_:). The DBC spec. still requires the BS_: field (it's typically left blank), but when it is filled out, it's not just a number, i.e. ```BS: 500```` like in your dbc file. It should take the format 
```
bit_timing = 'BS_:' [baudrate ':' BTR1 ',' BTR2 ] ; baudrate = unsigned_integer ;
BTR1 = unsigned_integer ;
BTR2 = unsigned_integer ;
```
2. Lines 38-41 where the attribute default values are set is missing a _. ```BA_DEF_DEF should be BA_DEF_DEF_```
3. I don't believe multi-line comments are supported like it's outlined in the template file. I couldn't find anything in the official Vector spec. stating that it was allowed.

If it's desirable, I can submit a pull request with the necessary corrections to get the template dbc file in a state where it can actually be parsed by CANDB++.

**Comment by stefanhoelzl** [comment]
Hello,

thank you for pointing out these issues!
Please go on and open a PR to fix theses issues!

Thanks!

**Comment by stefanhoelzl** [comment]
was fixed by #3
